rdf_differ/diff_getter.py

```diff
@@ -0,0 +1,242 @@
+        :return:
+            * datasetURI = dataset/scheme URI,
+            * version history graph = the graph that stores the structure of the calculated diffs
+            * current version graph = the graph that corresponds to the latest version of the dataset,
+            * datasetVersions = list of loaded dataset versions as declared by the datasets themselves,
+            * versionIds = list of versionIds as provided in the configurations file
+            * versionNamedGraphs = named graphs where the versions of datasets are loaded
+        """
+        query_result = self.execute_query(dataset_name=dataset_name,
+                                          sparql_query=SKOS_HISTORY_PREFIXES + QUERY_DATASET_DESCRIPTION)
+
+        return self._extract_dataset_description(response=query_result)
+
+    def execute_query(self, dataset_name: str, sparql_query: str) -> dict:
+        endpoint = SPARQLWrapper(self.make_sparql_endpoint(dataset_name=dataset_name))
+
+        endpoint.setQuery(sparql_query)
+        endpoint.setReturnFormat(JSON)
+        return endpoint.query().convert()
+
+    def list_datasets(self) -> List[str]:
+        response = requests.get(urllib.parse.urljoin(self.triplestore_service_url, "/$/datasets"),
+                                auth=HTTPBasicAuth('admin', 'admin'))
+
+        if response.status_code != 200:
+            raise Exception(f"Fuseki server request ({response.url}) got response {response.status_code}")
+
+        return self._select_dataset_names_from_fuseki_response(response=response)
+
+    def __init__(self, triplestore_service_url: str):
+        self.triplestore_service_url = triplestore_service_url
+
+    def make_sparql_endpoint(self, dataset_name: str):
+        return urllib.parse.urljoin(self.triplestore_service_url, dataset_name + "/sparql")
+
+    def _select_dataset_names_from_fuseki_response(self, response):
+        """
+            digging for the list of datasets
+        :param response: fuseki API response
+        :return:
+        """
+        result = json.loads(response.text)
+        return [d_item['ds.name'] for d_item in result['datasets']]
+
+    def _extract_dataset_description(self, response: dict) -> dict:
+        """
+            digging for:
+            * datasetURI = dataset/scheme URI,
+            * version history graph = the graph that stores the structure of the calculated diffs
+            * current version graph = the graph that corresponds to the latest version of the dataset,
+            * datasetVersions = list of loaded dataset versions as declared by the datasets themselves,
+            * versionIds = list of versionIds as provided in the configurations file
+            * versionNamedGraphs = named graphs where the versions of datasets are loaded
+        :param response: sparql query result
+        :return:
+        """
+        helper_current_version = [line['currentVersionGraph']['value'] for line in response['results']['bindings'] if
```

Comment:
I think `line` is not the best variable name for this case, as it seems that you're not iterating through lines but list items

_if you decide to change it, you should also do it for the next lines as well_